packages/cli-doctor/package.json

```diff
@@ -19,7 +19,6 @@
     "envinfo": "^7.10.0",
     "execa": "^5.0.0",
     "hermes-profile-transformer": "^0.0.6",
-    "ip": "^1.1.5",
```

Comment:
since it was in `package.json`, was it used anywhere in the `doctor` command?

Reply:
no, i guess it's leftover.